groups/go.mod

```diff
@@ -14,4 +14,18 @@ require (
 	k8s.io/test-infra v0.0.0-20191024183346-202cefeb6ff5
 )
 
+require (
+	github.com/clarketm/json v1.13.0 // indirect
+	github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e // indirect
+	github.com/golang/protobuf v1.3.5 // indirect
+	github.com/google/go-cmp v0.4.0 // indirect
+	github.com/googleapis/gax-go/v2 v2.0.5 // indirect
+	go.opencensus.io v0.22.3 // indirect
+	golang.org/x/sys v0.0.0-20200331124033-c3d80250170d // indirect
+	golang.org/x/text v0.3.2 // indirect
+	google.golang.org/appengine v1.6.5 // indirect
+	google.golang.org/grpc v1.28.0 // indirect
+	gopkg.in/yaml.v3 v3.0.0-20190709130402-674ba3eaed22 // indirect
```

Comment:
We need to have 2 versions of this package ? (I feel like the answer is yes because go modules)

Reply:
```
$ go mod why gopkg.in/yaml.v3
# gopkg.in/yaml.v3
k8s.io/k8s.io/groups
k8s.io/test-infra/pkg/genyaml
gopkg.in/yaml.v3
```

Reply:
Which is used for `go run reconcile.go --print` although to be fair the only thing is getting us at the moment is a preamble of 
```
# This file has the list of groups in kubernetes.io gsuite org that we use
# for granting permissions to various community resources.
```
before the groups are dumped out.  This is because the `GoogleGroup` type doesn't have any commented fields.

Reply:
I opted to depend directly on v3 in the reconciler, there weren't any breaking changes in our usage.

```
$ go mod why gopkg.in/yaml.v3
# gopkg.in/yaml.v3
k8s.io/k8s.io/groups
gopkg.in/yaml.v3
```